helmfile/charts/networkpolicy-common/templates/coredns/allow-coredns.yaml

```diff
@@ -0,0 +1,68 @@
+{{ if .Values.coredns.enabled }}
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: allow-coredns
+spec:
+  podSelector:
+    matchLabels:
+      k8s-app: kube-dns
+  policyTypes:
+    - Egress
+    - Ingress
+  ingress:
+    - ports:
+      - port: 53
+        protocol: UDP
+      - port: 53
+        protocol: TCP
+    - from:
+      - podSelector:
+          matchLabels:
+            k8s-app: kube-dns
+    - from:
+        - podSelector:
+            matchLabels:
+              app: prometheus
+          namespaceSelector:
+            matchLabels:
+              kubernetes.io/metadata.name: monitoring
+      ports:
+        - port: 9153
+  egress:
+    - to:
+      {{- range $IP := .Values.global.apiserver.ips }}
+      - ipBlock:
+            cidr: {{ $IP }}
+        {{- end }}
+      ports:
+        - port: {{ .Values.global.apiserver.port }}
+    - to:
+      {{- range $IP := .Values.global.nodes.ips }}
+      - ipBlock:
+            cidr: {{ $IP }}
+        {{- end }}
+      ports:
+        - port: 53
+          protocol: UDP
+        - port: 53
+          protocol: TCP
+    - to:
+      - ipBlock:
+          cidr: 10.233.0.3/32
```

Comment:
Even though this shouldn't change often I would still like this to be configurable in case of it being different somewhere